debian/source/format

```diff
@@ -0,0 +1 @@
+1.0
```

Comment:
Why not `3.0 (native)` instead?

Reply:
@sunweaver might have an answer to this one